internal/core/cluster/cluster.go

```diff
@@ -71,8 +71,16 @@ func NewCluster() (Cluster, error) {
 	})
 }
 
-func NewClusterWithDiscoveryTools(discoveryTools *DiscoveryTools) (Cluster, error) {
-	var cluster = Cluster{
+func NewClusterWithDiscoveryTools(discoveryTools *DiscoveryTools) (*Cluster, error) {
+	commandExecutor := utils.Executor{}
```

Comment:
Why have you changed the order of the things here?
Was something wrong?
Or is it simply to avoid the declaration of a not needed var?

Reply:
> Or is it simply to avoid the declaration of a not needed var?

this